pkg/golinters/lll/lll.go

```diff
@@ -121,7 +121,7 @@ func getLLLIssuesForFile(filename string, maxLineLen int, tabSpaces string) ([]r
 					Filename: filename,
 					Line:     lineNumber,
 				},
-				Text:       fmt.Sprintf("line is %d characters", lineLen),
+				Text:       fmt.Sprintf("The line is %d characters long, which exceeds the maximum of %d characters.", lineLen, maxLineLen),
```

Comment:
Usually linters warnings not ended with period?

Reply:
And not capitalized 🤔

Reply:
I think there is no convention about linter messages: some have a period, others do not, and the same thing applies to capitalization.